apps/system/style/tracking_notice/tracking_notice.css

```diff
@@ -0,0 +1,182 @@
+
+#tracking-notice {
+  display: block;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  background-color: rgba(199, 199, 199, 0.85);
+  padding: 2.2rem 1.5rem;
+  box-sizing: border-box;
+  pointer-events: all;
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 300ms, visibility 0s 300ms
+}
+
+#tracking-notice:not(.hidden) {
+  animation-name: gaia-dialog-entrance;
+  animation-duration: 300ms;
+  animation-timing-function: cubic-bezier(0.175, 0.885, 0.320, 1.275);
+  animation-fill-mode: forwards;
+  opacity: 1;
+  visibility: visible;
+  transition: opacity 300ms;
+}
+
+#tracking-notice > div.panel {
+  background-color: #f4f4f4;
+  width: 100%;
+  height: calc(100% - 6rem);
+  box-sizing: border-box;
+  padding: 1.5rem 1.5rem 0;
+}
+
+
+#tracking-notice header h1::before {
+  content: "";
+  width: 2.67rem;
+  height: 3.1rem;
+  position: absolute;
+  margin-left: -3.2rem;
+  background-size: contain;
+  background-image: url("images/tp_shield.png");
+  background-repeat: no-repeat;
+}
+
+#tracking-notice header {
+  border-bottom: solid 1px #e7e7e7;
+}
+
+#tracking-notice header h1 {
+  font-size: 1.8rem;
+  color: #858585;
+  font-weight: 500;
+  font-style: italic;
+  text-align: center;
+  line-height: 3.1rem;
+  margin-left: 2.5rem;
+  margin-top: 0;
+}
+
+#tracking-notice p {
+  font-size: 1.7rem;
+  font-style: italic;
+  font-weight: 400;
+  color: #4d4d4d;
+  margin-bottom: 2.8rem;
+}
+
+#tracking-notice a {
+  color: #0accf5;
+  text-decoration: underline;
+}
+
+
+#tracking-notice span {
+  font-size: 1.3rem;
+  font-weight: 400;
+  color: #4d4d4d;
+  position: relative;
+  top: -12%;
+  left: 18%;
+}
+
+#tracking-notice-image {
+  display: block;
+  width: 100%;
+  height: 38%;
+  background-size: contain;
+  margin: 0 auto;
+  background-image: url("images/tp_screen.png");
+  background-repeat: no-repeat;
+}
+
+#tracking-notice .setting {
+  position: relative;
+  top: .6rem;
+}
+
+#tracking-notice .setting > * {
```

Comment:
just spoke with :bz about this in #content. so, in practice this might not be too slow because we use ancestor filters to optimize it. but wildcards as the right most selector will cause all things under both .setting, and #tracking-notice to be restyled on every match, and so if either of these change a lot, we are adding a lot to the style matching work. again, this might not be noticeable to the user, but if we can avoid it we should.